eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/UtxoSpec.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
The module doesn't compile without it. It's needed because `logEntry` was changed to take a `Doc`.

I expect you see this as a further argument for introducing a new name rather than changing the type :smile: